docs/content/en/docs/reference/clusterspec/packages.md

```diff
@@ -0,0 +1,938 @@
+---
+title: "Packages configuration"
+linkTitle: "Packages configuration"
+weight: 10
```

Comment:
Would you mind changing the weight to 40? The other configuration files relate to provider configs and that will keep those all together in the left nav.